index.html

```diff
@@ -0,0 +1,847 @@
+                    <div class="second-info">
+                        <span class="money"> $18</span>
+                        <span class="series"> series b</span>
+                    </div>
+                    <div class="third-info">
+                        <p>Health and Wellness</p>
+                        <p>Second Alpha partners, +2</p>
+                    </div>
+                    <div class="forth-info">
+                        HealthCare.com Secure $18M in Series B Funding on FinSMEs
+                    </div>
+
+                </div>
+
+                <div class="funding-grid3">
+                    <div class="first-info">
+                        Healthcare.com
+                    </div>
+                    <div class="second-info">
+                        $18
+                    </div>
+                    <div class="third-info">
+                        <p>Health and Wellness</p>
+                        <p>Second Alpha partners, +2</p>
+                    </div>
+                    <div class="forth-info">
+                        HealthCare.com Secure $18M in Series B Funding on FinSMEs
+                    </div>
+
+                </div>
+
+                <div class="funding-grid4">
+                    <div class="first-info">
+                        Healthcare.com
+                    </div>
+                    <div class="second-info">
+                        $18
+                    </div>
+                    <div class="third-info">
+                        <p>Health and Wellness</p>
+                        <p>Second Alpha partners, +2</p>
+                    </div>
+                    <div class="forth-info">
+                        HealthCare.com Secure $18M in Series B Funding on FinSMEs
+                    </div>
+
+                </div>
+            </div>
+        </section>
+        <!--/////////////////////////////////this is the end of the section for funding/////-->
+        <div class="essai">
+            <section class="gear gear1">
+
+                <a href="#" class="gear-link">
+                    <div class="article-header"></div>
+                </a>
+                <h1><a href="#" class="article-titles">Gear ></a></h1>
+
+                <div class="img-gear img-gear1">
+                    <div class="text-link"><br><br><br>
+                        <h6>MOST POPURAL</h6>
+                        <p>
+                            Jack Dorsey:Twitter will stop all political
+                        </p>
+                        <small> ads next month</small>
+                    </div>
+                </div>
+                <div class="latest-middle latest-middle1">
+                    <div class="latest">
+                        <p>LATEST</p>
+                        <p>Notorious anon forum 8chan is back online as 8kun</p>
+                    </div>
+                    <div class="latest-img">
+                        <img src="images/icon1.png" alt="">
```

Comment:
##### Categories (Apps, GERA, Tech, etc) section

###### Medium version changes

* [ ] This section should not overflow ([link to page](https://thenextweb.com/))

**Yours**
<img width="1035" alt="Screenshot 2020-01-03 at 16 34 50" src="https://user-images.githubusercontent.com/31304943/71732368-5ad9cb80-2e47-11ea-80be-c8697118678e.png">

**Original**
<img width="1035" alt="Screenshot 2020-01-03 at 16 37 58" src="https://user-images.githubusercontent.com/31304943/71732394-6cbb6e80-2e47-11ea-8191-6baa78c69eb0.png">